src/components/InventoryTable/InventoryTable.js

```diff
@@ -349,4 +357,24 @@ InventoryTable.propTypes = {
   showNoGroupOption: PropTypes.bool, // group filter option
 };
 
-export default InventoryTable;
+const InventoryTableWrapper = forwardRef(
+  ({ isolateStore = false, ...props }, ref) => {
+    const store = useMemo(
+      () => (isolateStore ? getStore() : undefined),
```

Comment:
Do we want to limit the reducers to only those we use with InventoryTable? Something we discussed earlier. But it's more of "nice to have", so if you don't want to work on this, just leave the "TODO" note

Reply:
I think we will work on it in a different PR
I'll add a todo note like you suggested @gkarat

Reply:
What do you think about this note? 
`// TODO: Create a partial store containing only relevant Slices & Reducers relevant to the InventoryTable`
@gkarat